juego.py

```diff
@@ -58,8 +58,8 @@ def cargar_imagen(nombre,trasnparent=False):
         imagen = \
             pygame.transform.scale(imagen,
                                    (int(sizex * scale_x), int(sizey * scale_y)))
-     except pygame.error, message:
-          raise SystemExit, message
+     except pygame.error as message:
+          raise SystemExit(message)
```

Comment:
Is the `try..catch` necessary now? it looks like all we do is halt the program, so maybe we could remove it now.

Reply:
Yes it is necesaary; this function loads the images. If any image is not loaded then it returns an error but does not exit the program. I confirmed this by commenting out the `exception handling` part. Here is what I got:

```
Traceback (most recent call last):
  File "/usr/share/sugar/activities/jamath.activity/juego.py", line 286, in run
    level = self.main()
  File "/usr/share/sugar/activities/jamath.activity/juego.py", line 87, in main
    fondo = cargar_imagen('data/m.jpg')
  File "/usr/share/sugar/activities/jamath.activity/juego.py", line 56, in cargar_imagen
    imagen = pygame.image.load(nombre)
pygame.error: Couldn't open data/m.jpg
1603806363.047307 ERROR root: Event queue full!
1603806363.055937 ERROR root: Event queue full!
1603806363.069454 ERROR root: Event queue full!
1603806363.076200 ERROR root: Event queue full!
1603806363.083706 ERROR root: Event queue full!
```
And the image of what happens in the activity:
![JAMath](https://user-images.githubusercontent.com/46154968/97311200-1d3ca800-188a-11eb-8f95-92969c5042ec.png)
So it's logical to exit the activity if any image does not load.

Reply:
Makes sense. Thanks :smile: